### PR TITLE
Updated the sscheck api url from the ssm parameter

### PR DESCRIPTION
**Pull request by alexiswl**
* Also use 'currentVersion' attribute when specifying functionArns

Resolves  https://github.com/umccr/orcabus/issues/265

### CODE REVIEW

File: config/constants.ts [hunk 1]
```diff
@@ -99,6 +99,7 @@ Resources required for BaseSpace TES stack
  */
 
 export const basespaceAccessTokenSecretName = '/manual/BaseSpaceAccessTokenSecret'; // pragma: allowlist secret
+export const dataPortalApiUrlSsmParameterName = '/data_portal/backend/api_domain_name';
```
Comment (victorskl) [inline config/constants.ts:102]: Sorry Alexis, if I made you confuse...
This has to be SampleSheet check API domain parameter pointer.

I just check in SSM and, it is avail at this coordinate:

```
/sscheck/lambda-api-domain
```

```
export AWS_PROFILE=umccr-dev-admin
aws ssm get-parameter --name '/sscheck/lambda-api-domain' --output json | jq '.Parameter.Value'
```

And. Probably, all down under changes (vars, etc) have to re-align naming to reflect it is `samplesheet-check` or `sscheck` accordingly. 

It is NOT Portal API domain URL or domain name.

**Review by victorskl — COMMENTED** [review]
Need change to sscheck API domain URL.

File: config/constants.ts (hunk 1 above)
Comment (alexiswl) [reply config/constants.ts:102]: Ah my bad, will fix this later tonight

Comment (alexiswl) [reply config/constants.ts]: Done

**Comment by victorskl** [comment]
Reviewing this, give me 1 tick.

File: lib/workload/stateless/stacks/bs-runs-upload-manager/deploy/stack.ts [hunk 2]
```diff
@@ -46,6 +48,12 @@ export class BsRunsUploadManagerStack extends cdk.Stack {
       props.basespaceTokenSecretId
     );
 
+    const ss_check_api_ssm_parameter_obj = ssm.StringParameter.fromStringParameterName(
+      this,
+      'DataPortalApiUrlSsmParameter',
```
Comment (victorskl) [inline lib/workload/stateless/stacks/bs-runs-upload-manager/deploy/stack.ts:53]: Change to `'ssCheckApiUrlSsmParameter'` here too

File: lib/workload/stateless/stacks/bs-runs-upload-manager/deploy/stack.ts [hunk 3]
```diff
@@ -11,6 +12,7 @@ export interface BsRunsUploadManagerConfig {
   icaTokenSecretId: string; // IcaSecretsPortal
   portalTokenSecretId: string; // orcabus/token-service-jwt
   basespaceTokenSecretId: string; // /manual/BaseSpaceAccessTokenSecret
+  ssCheckApiUrlSsmParameterName: string; // /data_portal/backend/api_domain_name
```
Comment (victorskl) [inline lib/workload/stateless/stacks/bs-runs-upload-manager/deploy/stack.ts:15]: Need update comment

```
ssCheckApiUrlSsmParameterName: string; // /sscheck/lambda-api-domain
```

**Review by victorskl — APPROVED** [review]
LGTM. Just minor comment to attend. Then, merge whenever you ready.

**Pull request merged**